aip/general/0192.md

```diff
@@ -129,9 +134,9 @@ of inadvertent omissions of the internal content annotation.
 
 ## Changelog
```

Comment:
We should add an entry to the changelog noting the addition of the guidance around `deprecated = true`.

Reply:
Done.